draft-ietf-rats-architecture.md

```diff
@@ -842,40 +842,47 @@ a different format.
-signer's responsibility to ensure that the values are still correct when they
-are signed.  For example, values might have been generated at boot, and then
-used in claims as long as the signer can guarantee that they cannot have changed
-since boot.
+# Freshness
+
+A remote entity (Verifier or Relying Party) may need to learn the point in
+time (i.e., the "epoch") an Evidence or Attestation Result have been
+produced.
+This is essential in deciding whether the included Claims and their values can be considered fresh, meaning they still reflect the latest state of the Attester, and that any Attestation Result was generated using the latest Appraisal Policy for Evidence.
+
+Freshness is assessed based on a policy defined by the consuming entity,
+Verifier or Relying Party, that compares the estimated epoch against an
+"expiry" threshold defined locally to that policy.
+There is, however, always a race condition possible in that the state of the Attester, and the Appraisal Policy for Evidence, might change immediately after the Evidence or Attestation Result was generated.
+The goal is merely to narrow their recentness to something the Verifier (for Evidence) or Relying Party (for Attestation Result) is willing to accept.
+Freshness is a key component  for enabling caching and reuse of both Evidence and Attestation Results, which is especially valuable in cases where their computation uses a substantial part of the resource budget (e.g., energy in constrained devices).
+
+There are two common approaches for determining the epoch of an Evidence or Attestation Result.
+
+<!-- Explicit Timekeeping using Synchronized Clocks -->
+
+The first approach is to rely on synchronized and trustworthy clocks, and include a signed timestamp (see {{?I-D.birkholz-rats-tuda}}) along with the Claims in the Evidence or Attestation Result.
+Timestamps can also be added on a per-Claim basis if wanted, to allow distinguishing the time of creation of Evidence from the time that a specific Claim value was generated.
+Clocks trustworthiness typically requires additional Claims about the signer's time synchronization mechanism.
+
+<!-- Implicit Timekeeping using Nonces -->
+
+A second approach places the onus of timekeeping solely on the appraising entity (e.g., the Verifier for Evidence, or the Relying Party for an Attestation Result).
+This might be an appropriate choice in case the Attester does not have a reliable clock or time synchronisation is otherwise impaired.
+In this approach, a non-predictable nonce is generated by the appraising entity, and the nonce is then signed and included along with the Claims in the Evidence or Attestation Result.
+After checking the sent and received nonces are the same, the appraising entity knows that the Claims were signed after the nonce was generated, effectively locking its clock to the produced payload.
+This allows associating a "rough" epoch to the Evidence or Attestation Result.
```

Comment:
Replace "This" with "Timekeeping" ?

Reply:
Not sure: "this" is a bit more than just "timekeeping", it refers to "an approach that places the onus of timekeeping solely on the appraising entity".

Reply:
Maybe substitute your text in place of 'this' since the pronoun reference seems ambiguous?

Reply:
That seems a bit redundant.  I'd like to achieve some kind of compression - obviously without introducing ambiguity.  Let me see if I can concoct something decent.

Reply:
Ned: please check the proposal in 20e3d7